content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -101,13 +108,37 @@ See the Google Kubernetes Engine (GKE) guide to
 
 1. Check the [Kubeflow Pipelines GitHub repository](https://github.com/kubeflow/pipelines/releases) for available releases.
 
-1. Upgrade to a version of Kubeflow Pipelines standalone:
+1. Upgrade to a version of Kubeflow Pipelines standalone is the same as deploying that version:
 
      ```
      export PIPELINE_VERSION=<version-you-want-to-upgrade-to>
+     kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=$PIPELINE_VERSION
+     kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
      kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
      ```
 
+     This is for 0.4.0 and above, commands for older versions can be found at [Deploying Kubeflow Pipelines](#deploying-kubeflow-pipelines)
+
+1. Delete removed resources manually
```

Comment:
I would recommend changing "removed" to "obsolete". Also, please add a full stop at the end of the sentence.

Delete obsolete resources manually.

Reply:
thanks, added